src/discord-api.ts

```diff
@@ -141,3 +162,63 @@ const fetchData = async (url: string, token: string) =>
 			Accept: 'application/json',
 		},
 	})
+
+export const fetchAttachment = async (ctx: Context) => {
+	const request = ctx.req
+	const env = ctx.env
+
+	const origin = 'https://cdn.discordapp.com'
```

Comment:
You could put this in a const outside of this function if you want, and call it `DISCORD_ORIGIN`